### ISSUE TITLE
False Negative in Style/IndentationWidth-Cop

### ISSUE THREAD

**Issue by fonzerelly** [rubocop/rubocop#1165]
The Style/IndendationWidth-Cop seems to have Problems in rescue clauses. When I write this code:

``` ruby
def my_func
  puts 'do something outside block'
  begin
    puts 'do something error prone'
  rescue
     puts 'wrongly intended error handling'
  end
end
```

I did not configure this cop yet and if I use wrong indentation in the begin-clause, rubocop informs be about that error. But it seems to oversee it in the rescueblock.
I use rubocop 0.23.0
